joomla/joomla-cms
# [4.0] New feature. Adding a multilingual sample data plugin

### Summary of Changes
Adding a Multilingual Sample Data plugin

### Testing Instructions
Discover the plugin and enable it.
It will display in the Sample Data module in CPanel.
Make sure you do not have any sample data installed on your test site.
Make sure you have at least 2 languages installed and their Content Languages.
Launch the Multilingual Sample Data install

### Expected result
The plugin will implement the 3.x multilang installation which could be done at Joomla install time and is no more available in 4.0 (although the code is still present in the installation folder and, I guess, should be deleted.).
It uses similar methods and obtains the same results.

It enables the Language Filter system plugin.
It creates a Language Switcher module.
It creates one Main Menu per language with a Blog menu item as Home + a hidden List All Categories menu item.
It creates the corresponding menu modules.
It creates one article category per language and one article tagged to the same language in each category.
Associations are set for menu items, categories and articles.

<img width="969" alt="screen shot 2018-06-11 at 08 18 13" src="https://user-images.githubusercontent.com/869724/41215369-4cd9ac9e-6d50-11e8-974d-2fe7c2dbb53f.png">

<img width="924" alt="screen shot 2018-06-11 at 08 19 49" src="https://user-images.githubusercontent.com/869724/41215388-5c44eb62-6d50-11e8-961f-9ec453d00875.png">
